examples/svelte-apollo/nhost/migrations/default/1651998939184_create_table_public_author/up.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE "public"."author" (
```

Comment:
I would suggest changing the table name to plural as the table contains multiple authors.

Reply:
Done